server/server.js

```diff
@@ -20,9 +20,29 @@ server.listen(config.port);
 log.info('Server is on bruh and running on port: '+ config.port);
 
 //----------------------------------
+// Allow Cors Domain:
```

Comment:
I feel that having this type of documentation in the code isn't a great practice so it might be worth updating the readme file or something along those lines to explain this block of code rather than doing it through comments.